.gitignore

```diff
@@ -1,10 +1,7 @@
-# NPM
 /node_modules/
-npm-debug.log
```

Comment:
The rebase went fine however you still have the commit on your branch which changes the _.gitignore_ file. Please delete this commit from your branch and adjust the commit message of your actual commit.
Here is how you could do this:

``` bash
git checkout market-data-bretten
git reset HEAD~2 // Undo the last two commits, keep file changes
git checkout .gitignore // Throws away your changes on that file
git add . // Add all changed and new files
git commit -m "Add market data for Bretten."
git push --force origin market-data-bretten // Update your branch on GitHub
```